src/Illuminate/Queue/QueueManager.php

```diff
@@ -233,6 +234,16 @@ public function setDefaultDriver($name)
     }
 
     /**
+     * Get the name of the queue prefix.
+     *
+     * @return string
+     */
+    public function getQueuePrefix()
+    {
+        return isset($this->app['config']['queue.prefix']) ? $this->app['config']['queue.prefix'] : null;
```

Comment:
I see what's going on. In an *actual* application it returns the config repository, but in the *test* it is returning a simple array.  I've made @taylorotwell 's suggested change and updated the tests to use a config `Repository` rather than just the array.